sbnobj/Common/POTAccounting/BNBSpillInfo.h

```diff
@@ -0,0 +1,63 @@
+#ifndef sbncode_BNBSpillInfo_HH
+#define sbncode_BNBSpillInfo_HH
+
+#include <vector>
+
+namespace sbn {
+  class BNBSpillInfo {
+  public:
+    //typedef enum triggertype { kBNB, kEXT, kUnbiased, kNUMI } TriggerType; //!< Which beamline are we looking at
+    
+    unsigned long int spill_time_s; //!< The IFDB Beam Spill Time, unit sec
+    unsigned long int spill_time_ns; //!< The IFDB Beam Spill Time, unit nsec
+ 
+    //BeamQuaility 
+    //quality//EXT
+    //int EXTGates;
+
+    //quality//BNB
+    
+    double TOR860; //!< Toroid before Mag 875, units e12 Protons
+    double TOR875; //!< Toroid after Mag 875, units e12 Protons
+    
+    double LM875A; //!< Loss Monitor before the RWM, unit R/s
+    double LM875B; //!< Loss Monitor after the RWM, unit R/s
+    double LM875C; //!< Loss Monitor after the RWM, unit R/s
+    double HP875; //!< Horizontal Position Monitor after Mag 875, units mm
+    double VP875; //!< Verticle Position Monitor after Mag 875, units mm
+
+    double HPTG1; //!< Horizontal Position Monitor at Target Station 1, units mm 
+    double VPTG1; //!< Horizontal Position Monitor at Target Station 1, units mm
+
+    double HPTG2; //!< Horizontal Position Monitor at Target Station 2, closest to target, units mm
+    double VPTG2; //!< Horizontal Position Monitor at Target Station 2, closest to target, units mm
+    
+    double BTJT2; //!< Temperature of air exiting target, units Deg C
+
+    double THCURR; //!< Current applied to Horn, units kiloAmperes 
```

Comment:
I actually doubt that any of these measures would not be fitting in a single precision number (6-7 significant digits).
Unless there is some large offset, that is (like measuring the position of the beam with the location of the Director's office as reference).
Moving to single precision would save quite some memory, and I _suggest_ this to be done if feasible.

Reply:
switched to floats